### PR TITLE
[Minor][one line change] Fix an error message about start and end time validation. EOM

### PR DESCRIPTION
**Pull request by celian-garcia**

### CODE REVIEW

File: api/v2/api.go [hunk 1]
```diff
@@ -723,7 +723,7 @@ func (api *API) postSilencesHandler(params silence_ops.PostSilencesParams) middl
 	}
 
 	if sil.StartsAt.After(sil.EndsAt) || sil.StartsAt.Equal(sil.EndsAt) {
-		msg := "failed to create silence: start time must be equal or after end time"
+		msg := "failed to create silence: start time must neither be equal nor after end time"
```
Comment (simonpasquier) [inline api/v2/api.go]: Sorry for the late review. Can you change the sentence to be affirmative? `start time must be greater than end time`

Comment (celian-garcia) [reply api/v2/api.go]: Absolutely I'll do it now

**Comment by celian-garcia** [comment]
Mmh  I am sorry I don't get the circle ci issue... Any change that restart the workflow will solve the problem ?

**Comment by simonpasquier** [comment]
I've kicked the CI again.

**Comment by celian-garcia** [comment]
Ah I'm just realizing that your suggestion is still wrong... If we want to be affirmative let's change it to :

start time must be before the end time

"Greater than" with time is quite confusing. 

Would you mind if I create another PR ? I can't reopen this one

**Comment by simonpasquier** [comment]
Oops I've been too quick merging it. Please open a new PR.

**Pull request merged**